packages/theme/src/defaultTheme.js

```diff
@@ -24,6 +24,19 @@ const theme = {
     fill: ${palette.black};
     justify-content: center;
   `,
+  Card: () => ({ hasShadow, hasBorder }) => css`
```

Comment:
There is a story to move these to be co-located with the component, but this is fine until that story is played. Looks great!

Reply:
Oh wow that was the major feedback I had -- we should co-locate these!